lambdatimes/src/components/Header.js

```diff
@@ -1,15 +1,55 @@
 import React from 'react';
-
+import styled, {css} from 'styled-components'
 // Refactor this component to use styled components and not classNames. 
```

Comment:
you can skip the named {css} import from styled-components in this one because you are not using it in the styles, just the styled